analysis/stat_tests.py

```diff
@@ -48,15 +51,11 @@ def test_pair(measurements_x, measurements_y):
     for f_i in fuzzers:
         row = []
         for f_j in fuzzers:
-            if f_i == f_j:
-                # TODO(lszekeres): With Pandas 1.0.0+, switch to:
-                # p_value = pd.NA
-                p_value = np.nan
-            elif set(samples[f_i]) == set(samples[f_j]):
-                p_value = np.nan
-            else:
-                p_value = test_pair(samples[f_i], samples[f_j])
-            row.append(p_value)
+            value = np.nan
```

Comment:
I'll take another look at this.  I had a problem with the heatmap plot when trying to use `pd.NA`.

Reply:
`pd.NA` still causes problems here because the seaborn heatmap casts the data as a `float` type, which results in an error.  
```
  File "/home/user/.virtualenvs/venv_plot/lib/python3.8/site-packages/seaborn/matrix.py", line 189, in _determine_cmap_params
    calc_data = plot_data.astype(float).filled(np.nan)
TypeError: float() argument must be a string or a number, not 'NAType'
```

Reply:
got it, thanks for checking!